soccer/radio/SimRadio.cpp

```diff
@@ -15,15 +15,22 @@
 
 using namespace std;
 using namespace Packet;
-
-static QHostAddress LocalAddress(QHostAddress::LocalHost);
+using namespace boost::asio;
 
 SimRadio::SimRadio(Context* const context, bool blueTeam)
-    : _context(context), _blueTeam(blueTeam) {
-    switchTeam(blueTeam);
+    : _context(context),
+      _blueTeam(blueTeam),
+      _socket(_io_service, ip::udp::endpoint(ip::udp::v4(),
+                                             blueTeam ? SimBlueStatusPort
+                                                      : SimYellowStatusPort)) {
+    _grsim_endpoint = ip::udp::endpoint(ip::udp::v4(), SimCommandPort);
+
+    // TODO: Make sure the buffer is big enough.
```

Comment:
Can we get an issue # for this?

Reply:
Gah, left this in while I was working and forgot to delete it. The return packets from grSim are actually only a byte long in this version of the protocol (in later versions it's a proper protobuf, so a lot of this is about to be rewritten when we upgrade grSim versions).